examples/surrounding_nitrogen_bonds/n_n_fingerprint_funcs.py

```diff
@@ -0,0 +1,50 @@
+"""
+Various functions to create fingerprints for single nitrogen-nitrogen bonds
+"""
+
+from openeye import oechem, oeomega, oequacpac
+
+def find_neighboring_atoms(mol: oechem.OEMol, match: oechem.OEMatchBaseIter):
+    """
+    Finds and returns the neighbor atoms of a nitrogen-nitrogen bond as a list
+    given a match
+    """
+    neighbors = []
+    
+    for trgtBond in match.Target().GetTargetBonds():
+        trgtIdxs = (trgtBond.GetIdx()-1, trgtBond.GetIdx())
+        for atom in mol.GetAtoms():
+            if atom.GetIdx() in trgtIdxs:
+                for bond in atom.GetBonds():
+                    nbor = bond.GetNbr(atom)
+                    if nbor.GetIdx() not in trgtIdxs:
+                        neighbors.append(nbor.GetAtomicNum())
+    return neighbors
+
+def wiberg_bond_order(mol: oechem.OEMol, match: oechem.OEMatchBaseIter):
+    """
+    Finds and returns a list of Wiberg bond orders of nitrogen-nitrogen bonds
+    given a match
+    """
+    wbo = []
+    AM1_CALCULATOR = oequacpac.OEAM1()
```

Comment:
I think you can define `AM1_CALCULATOR` and `omega` as global constants, but it probably won't make any difference performance-wise since most of the computation is spent on line 42 anyway. Either way is fine with me.